src/main/java/io/strimzi/kafka/bridge/config/ConfigRetriever.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge.config;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.stream.Collectors;
+
+/**
+ * Retrieve the bridge configuration from properties file and environment variables
+ */
+public class ConfigRetriever {
+
+    /**
+     * Retrieve the bridge configuration from the properties file provided as parameter
+     * and adding environment variables
+     * If a parameter is defined in both properties file and environment variables, the latter wins
+     *
+     * @param path path to the properties file
+     * @return configuration as key-value pairs
+     * @throws IOException when not possible to get the properties file
+     */
+    public static Map<String, Object> getConfig(String path) throws IOException {
+        Map<String, Object> configuration;
+        try (InputStream is = new FileInputStream(path)) {
+            Properties props = new Properties();
+            props.load(is);
+            configuration =
+                    props.entrySet().stream().collect(
+                            Collectors.toMap(
+                                    e -> String.valueOf(e.getKey()),
+                                    e -> String.valueOf(e.getValue()),
+                                    (prev, next) -> next, HashMap::new
+                            ));
+        }
+        configuration.putAll(System.getenv());
```

Comment:
Should it do some filtering for some prefixes we support here? Or do we really want to accept everything? Also, should it follow some commonly used conversions between env. vars and properties? E.g. `HTTP_PORT` to `http.port`? Or do you expect the env. vars to be named the same as the properties?

Reply:
Replicating the same behaviour of Vert.x Config, it's just loading everything and there is no convention about names, you have to define the env var as the same name of the property to override (so no `HTTP_PORT` to `http.port`). The filtering happens later when we build the actual config.
Anyway, while doing this I was not sure to continue to support env vars even because we don't use them anyway and everything is configured via properties file in the bridge.

Reply:
> Anyway, while doing this I was not sure to continue to support env vars even because we don't use them anyway and everything is configured via properties file in the bridge.

Might deserve a wider discussion and checking with users if they use it or not. But in general yes, it might make sense.